src/framework/shortcuts/qml/MuseScore/Shortcuts/ShortcutsPage.qml

```diff
@@ -74,6 +74,7 @@ Item {
 
         function startEditCurrentShortcut() {
             editShortcutDialog.startEdit(shortcutsModel.currentShortcut, shortcutsModel.shortcuts())
+            editShortcutDialog.currentShortcut = shortcutsModel.currentShortcut
```

Comment:
I think it is preferable to move this line to EditShortcutDialog.startEdit.

Or, maybe even better, get rid of the `currentShortcut` property and instead add a Q_PROPERTY to `EditShortcutModel`, and use that in QML.